LICENSE

```diff
@@ -6,7 +6,7 @@ Permission is hereby granted, free of charge, to any person obtaining a copy
 of this software and associated documentation files (the "Software"), to deal
```

Comment:
Just FYI, 
1. 'persons' is a term that can be used in legal writing. This license is legal writing, so it's okay to put it as 'persons'. 
2. Don't change licenses or copyright notices.

Reply:
i mass changed the word person i think so it changes the license too ill added later

Reply:
Noted on this.